### DISCUSSION TITLE
Dreame A1 Mower

### DISCUSSION THREAD

**Discussion by Sweti934** [Tasshack/dreame-vacuum#560] [Ideas]
Hi,

I - currently - don´t own a Dreame A1 lawn mowing robot. However, could this one also be supported in here? This would also be really really awesome! Thanks a lot!

**Comment by Tasshack** [comment]
I don't own one too so it won't be supported in this integration.

**Reply by jszkiela72** [reply]
I have a Dreame A1. Is there anything I can do to get you to consider adding it to your integration?

**Reply by Tasshack** [reply]
You can buy me a one.
